libs/core/src/lib/popover/popover-control/popover-control.component.scss

```diff
@@ -0,0 +1,3 @@
+.fd-popover-outline:focus-visible {
+    outline: var(--sapContent_FocusWidth) dotted var(--sapContent_FocusColor)
+}
```

Comment:
this should be moved in fund-styles

Reply:
https://github.com/SAP/fundamental-styles/pull/2555